src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -177,6 +178,8 @@ export default class OrganizationDiscover extends React.Component {
       label: name,
     }));
 
+    console.log('query is...', query);
```

Comment:
I think you have to rebase & delete these console logs.

Reply:
Will do